releasenotes/index.md

```diff
@@ -6,6 +6,7 @@ permalink: /releasenotes/
 
 The Release Notes and Javadocs for every major release of Apache jclouds are listed chronologically below.
 
+* [1.9.2](/releasenotes/1.9.2) | [Javadoc](/reference/javadoc/1.9.x/)
```

Comment:
> Should we deploy all missing versions?

Good question. I think we have a couple of options here:

* Point the A.B.x Javadoc link to A.B.0, which seems to be the case with 1.9.x now. But then we should probably change the link from `A.B.x` to `A.B.0`?
* Update the A.B.x Javadoc to point to the latest release, i.e. overwrite the 1.9.0 Javadoc with 1.9.2, and keep the `1.9.x` link. This seems most consistent with the name of the link
* Publish Javadoc for all the versions, and then change the links accordingly

@nacx What do you think..?